spec/unit/equalizer/universal_spec.rb

```diff
@@ -146,6 +146,22 @@ def initialize(firstname, lastname)
         expect(instance.hash)
           .to eql([firstname, lastname, klass].hash)
       end
+
+      context 'when immutable' do
```

Comment:
Are you implying we should define a shared context for running under both `with keys` and `when immutable`? Or just defining `describe '#hash'` under `when immutable`?